docs/config.md

```diff
@@ -1404,6 +1408,39 @@ Default: `"5m"`
 
 Type: `duration`
 
+## `Provider`
+
+Configuration applied to the initial one-time announcement of fresh CIDs
+created with `ipfs add`, `ipfs files`, `ipfs dag import`, `ipfs block|dag put`
+commands.
+
+For periodical DHT reprovide settings, see [`Reprovide.*`](#reprovider).
+
+### `Provider.Strategy`
+
+Legacy, not used at the moment, see [`Reprovider.Strategy`](#reproviderstrategy) instead.
```

Comment:
```suggestion
Legacy, no longer used, see [`Reprovider.Strategy`](#reproviderstrategy) instead.
```
Otherwise, it sounds like we are trying to communicate a planned future use.

Reply:
We might? Now that we have separate queues, we may have strategies related to `provide` that are different than `reprovide` 

Left as-is for now.